doc/classes/TreeItem.xml

```diff
@@ -226,11 +226,19 @@
 				Returns the next sibling TreeItem in the tree or a null object if there is none.
 			</description>
 		</method>
+		<method name="get_next_in_tree">
+			<return type="TreeItem" />
+			<param index="0" name="wrap" type="bool" default="false" />
+			<description>
+				Returns the next TreeItem in the tree (in the context of a Depth-first search) or a null object if there is none.
```

Comment:
```suggestion
				Returns the next TreeItem in the tree (in the context of a depth-first search) or a null object if there is none.
```
Same below.

Reply:
This was not fully addressed. While `TreeItem` without braces is ok as is, depth-first search should not be capitalized.

Reply:
Fixed.